src/docs/wasm-compilation-pipeline.md

```diff
@@ -7,19 +7,23 @@ WebAssembly is a binary format that allows you to run code from programming lang
 
 ## Liftoff
 
-V8 first compiles a WebAssembly module with its baseline compiler, [Liftoff](/blog/liftoff). Liftoff is a [one-pass compiler](https://en.wikipedia.org/wiki/One-pass_compiler), which means it iterates over the WebAssembly code once and emits machine code immediately for each WebAssembly instruction. One-pass compilers excel at fast code generation, but can only apply a small set of optimizations. Indeed, Liftoff can compile WebAssembly code very fast, 10s of megabytes per second. Once Liftoff compilation is finished, the compiled WebAssembly module is returned to JavaScript.
+Initially, V8 does not compile any functions in a WebAssembly module. Instead, functions get compiled lazily with the baseline compiler [Liftoff](/blog/liftoff) when the function gets called for the first time. Liftoff is a [one-pass compiler](https://en.wikipedia.org/wiki/One-pass_compiler), which means it iterates over the WebAssembly code once and emits machine code immediately for each WebAssembly instruction. One-pass compilers excel at fast code generation, but can only apply a small set of optimizations. Indeed, Liftoff can compile WebAssembly code very fast, 10s of megabytes per second.
```

Comment:
Maybe change to "tens of megabytes" because "10s" looks like 10 seconds. I know that wasn't anything you changed, but I think it's more udnerstandable.

Reply:
Done.